ubiquibot-config-default.yml

```diff
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
```

Comment:
I wonder if it makes sense to parse the integer in the name and then derive its `weight` from it. This could be a new bounty if that makes sense to do. 

So in this case we would shift them all up by one, meaning, `Priority: 1 (Normal)` would be the first one.

Reply:
I guess most partners will stick to their own prioritization techniques (`Must have, Could have, High impact, Low impact, etc...`) so parsing an integer from a string is really useful only in our case

Reply:
That's why I have the name in the parenthesis e.g. Normal, Medium, High, Urgent, Emergency.

This is designed to be plain text/human readable. 

Easy to do it like this:

```
Priority: 1 (Low Impact)
Priority: 2 (Could Have) 
Priority: 3. (Must Have)
Priority: 4. (High Impact)
```